Library/Formula/curl.rb

```diff
@@ -45,6 +45,12 @@ class Curl < Formula
   depends_on "libressl" => :optional
   depends_on "nghttp2" => :optional
 
+  # This patch fix compile against LibreSSL. From:
```

Comment:
Oops. It was originally `these patches fix compile` but I changed it to a single patch and forgot the `fix` :smile:.